support-frontend/assets/helpers/redux/checkout/personalDetails/contributionsSideEffects.ts

```diff
@@ -0,0 +1,55 @@
+import type { ContributionsStartListening } from 'helpers/redux/contributionsStore';
+import * as storage from 'helpers/storage/storage';
+import { enableOrDisableForm } from 'pages/contributions-landing/checkoutFormIsSubmittableActions';
+import {
+	setEmail,
+	setFirstName,
+	setLastName,
+	setUserTypeFromIdentityResponse,
+} from './actions';
+
+export function addPersonalDetailsSideEffects(
+	startListening: ContributionsStartListening,
+): void {
+	startListening({
+		actionCreator: setFirstName,
+		effect(_, listenerApi) {
+			enableOrDisableForm()(
```

Comment:
:hand: It looks like `enableOrDisableForm` is a standard Redux thunk.  As such, the right way to use it is:

```js
dispatch(enableOrDisableForm())
```

and then the thunk middleware intercepts the thunk function and actually calls it with `dispatch` and `getState` as arguments:

https://redux.js.org/usage/writing-logic-thunks#thunk-usage-patterns

(also you would never need to do `getState.bind()` - it's just a simple getter function, not anything that relies on `this`)